flow/alerting/classifier.go

```diff
@@ -449,14 +452,21 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 				(strings.HasPrefix(pgErr.Message, "could not stat file ") &&
 					strings.HasSuffix(pgErr.Message, "Stale file handle")) ||
 				// Below error is transient and Aurora Specific
-				(strings.HasPrefix(pgErr.Message, "Internal error encountered during logical decoding")) ||
+				(strings.HasPrefix(pgErr.Message, "Internal error encountered during logical decoding")) &&
```

Comment:
(it looks like previously when this error was encountered it was `TableVolumeChangelogReaderReadNextChangelog` and did not have a hint to adjust config, so leaving it here)